docs/install/collecting-metrics/README.md

```diff
@@ -0,0 +1,97 @@
+This document describes how to set up the
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/about/) to
```

Comment:
Link seems dead, an option is `https://opentelemetry.io/docs/collector/`

Reply:
Thanks, fixed.